test/gurka/gurka.h

```diff
@@ -69,7 +69,9 @@ using nodelayout = std::unordered_map<std::string, midgard::PointLL>;
 
 namespace detail {
 
-boost::property_tree::ptree build_config(const std::string& tiledir) {
+boost::property_tree::ptree
+build_config(const std::string& tiledir,
+             const std::unordered_map<std::string, std::string>& config_options) {
```

Comment:
its convenient to be able to override bits of the config that gurka builds so i added that. originally i needed it for timezone information but in the end i was able to work around it. this will be useful for others though i reckon